solaris/solaris10/preremove.sh

```diff
@@ -4,7 +4,11 @@
 OSSEC_INIT="/etc/ossec-init.conf"
 control_binary="wazuh-control"
 
-. ${OSSEC_INIT}
+if [ -f ${OSSEC_INIT} ]; then
+  . ${OSSEC_INIT}
+else
+  VERSION=`/var/ossec/bin/${control_binary} -v`
+fi
 
 set_control_binary() {
```

Comment:
We can make here the same improvement as in **uninstall.sh** and just check if **wazuh-control** exists to decide if we call **wazuh-control** or **ossec-control**. Avoiding the first check of **ossec-init.conf**

Reply:
In this case, that improvement is not appropriate. The reason why is because the final goal is to determine the Wazuh version. So, for old versions, we need to make use of `ossec-init.conf` because `ossec-control` does not provide that information.

Reply:
But the VERSION is only used to set which control binary use in set_control_binary (ossec-control or wazuh-control), so, improving that logic, we will not need the VERSION anymore.

Reply:
Fixed